File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.js
class Solution {
    productExceptSelf(nums) {
        let output=[]
        output[0]=1
        let product=1
        for(let i=1;i<nums.length;i++){
            output[i]=output[i-1]*nums[i-1]
        }
        for(let i=nums.length-1;i>0;i--){
            product*=nums[i]
            output[i-1]*=product
        }
        return output
    }
}

File: Data Structures & Algorithms/majority-element/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        freqMap={}
        for num in nums:
            freqMap[num]=freqMap.get(num,0)+1
        for key,value in freqMap.items():
            if value>=len(nums)//2:
                return key